packages/components/organisms/f-checkout/src/store/analytics.module.js

```diff
@@ -0,0 +1,131 @@
+import Trak from '@justeat/f-trak';
+import { mapAnalyticsFieldNames } from '../services/mapper';
+
+import {
+    UPDATE_ANALYTICS_STATE,
+    UPDATE_CHANGED_FIELDS,
+    UPDATE_AUTOFILL
+} from './mutation-types';
+
+export default {
+    namespaced: true,
+
+    state: () => ({
+        serviceType: '',
+        restaurantId: '',
+        basket: {
+            id: '',
+            total: 0
+        },
+        isLoggedIn: false,
+        changedFields: [],
+        autofill: []
+    }),
+
+    actions: {
+        updateState ({ rootState, commit }, payload) {
+            const checkoutState = {
+                isLoggedIn: rootState.checkout.isLoggedIn,
+                basket: rootState.checkout.basket,
+                restaurantId: rootState.checkout.restaurantId,
+                serviceType: rootState.checkout.serviceType
+            };
```

Comment:
Related to my comments above, I don't think there's a need for this. We're just duplicating state here rather than referencing the `checkout` module directly.